src/index.ts

```diff
@@ -480,13 +480,15 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     }
 
     const url = `${configurations.baseUrl}/stream?cluster=${clusterIdentifier}`
-    streamer(eventBus, configurations, url, apiKey, standardHeaders, event => {
+
+    eventSource = new Streamer(eventBus, configurations, url, apiKey, standardHeaders, event => {
```

Comment:
You'll need to declare this variable, or else it'll get created on the global scope.
```suggestion
    const eventSource = new Streamer(eventBus, configurations, url, apiKey, standardHeaders, event => {
```

Reply:
just checked it might be ok, adding const give me a shadow warning. eventSource was reused from the previous code

<img width="703" alt="image" src="https://github.com/harness/ff-javascript-client-sdk/assets/115623869/4aa54857-5ae5-4330-8387-342a14e1ae2d">

Reply:
Oh right... I thought that was yanked out with the removal of the library. All good 👍🏻